exporters/requirements-dev.txt

```diff
@@ -1,9 +1,6 @@
 # Needed by exporters/tests
 pytest
-
-# Mentioned in the docs/Development.md
-# Also used by the github unittests.yml workflow
-coverage
+pytest-cov
```

Comment:
to avoid duplication of writing `coverage run -m ... && coverage report` everytime